packages/bpk-scrim-utils/src/withScrimmedPortal-test.tsx

```diff
@@ -108,44 +110,47 @@ describe('withScrimmedPortal', () => {
   });
 
   it('renders the wrapped component inside a portal correctly when runOnServer is false (default)', () => {
-    const DialogContent = () => <div>Dialog content</div>;
+    const DialogContent = () => <div data-testid="dialog-content">Dialog content</div>;
```

Comment:
Reading the tests it more evident that the prop name is confusing, we say runOnServer and we are checking that it renders on client, maybe we could use `ssrCompatible` or similar. Still if we could make the component self aware of the environment and avoid passing the prop it would be even better.